scripts/collect_ticks.py

```
@@ -0,0 +1,296 @@
+        except Exception as e:
+            db = {"bids": {}, "asks": {}, "best_bid": None,
+                  "best_ask": None, "malformed": 0, "err": f"down:{e}"}
+            db_err = db["err"]
+        time.sleep(random.uniform(0.0, JITTER_SEC))
+
+        tape: list[dict] = []
+        tape_err = ""
+        try:
+            tape_map = recent_trades(cid, w["seen_tape"], limit=TAPE_LIMIT)
+            tape = list(tape_map.get(w["up_token"], {}).items())
+        except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)